scheduler/management/commands/trigger_periodic_task.py

```diff
@@ -0,0 +1,76 @@
+from django.core.management import BaseCommand, CommandError
+from djcelery.models import PeriodicTask
+from djcelery.schedulers import ModelEntry
+
+
+class Command(BaseCommand):
+    help = ('Fire djcelery PeriodicTask manually. \n\n'
+            '* Only use this is things went wrong *')
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'periodic-task-id', type=int,
+            help='What is the PK of the PeriodicTask to fire manually')
+        parser.add_argument(
+            '--confirm', action='store_true', default=False,
+            help=('Do not ask for any kind of confirmation, '
+                  'I know what I am doing'))
+        parser.add_argument(
+            '--ignore-result', action='store_true', default=False,
+            help=('Do not wait for the task to complete and return '
+                  'the result'))
+        parser.add_argument(
+            '--timeout', type=int, default=60,
+            help=('How long to wait in secods for the result to return, '
+                  'set to 0 to disable. Defaults to 60 seconds.'))
+
+    def handle(self, *args, **options):
+        periodic_task_id = options['periodic-task-id']
+        ignore_result = options['ignore_result']
+        if options['timeout']:
+            timeout = options['timeout']
+        else:
+            timeout = None
+
+        def confirm(prompt):
+            if options['confirm']:
+                return True
+            try:
+                return raw_input(
```

Comment:
This won't work on Python 3. Django does include a six util for this though:

from django.utils.six.moves import input